user/models.py

```diff
@@ -0,0 +1,76 @@
+from django.db import models
+from django.core.mail import send_mail
+from django.contrib.auth.models import PermissionsMixin
+from django.contrib.auth.base_user import AbstractBaseUser
+from django.utils import timezone
+from django.contrib.auth.base_user import BaseUserManager
+
+"""
+https://github.com/django/django/blob/main/django/contrib/auth/models.py
+"""
+
+
+class UserManager(BaseUserManager):
+
+    def _create_user(self, email, password, **extra_fields):
+
+        if not email:
+            raise ValueError('The given email must be set')
+        email = self.normalize_email(email)  # 正規化
+        user = self.model(email=email, **extra_fields)
+        user.set_password(password)
+        user.save(using=self._db)
+        return user
+
+    def create_user(self, email, password=None, **extra_fields):
+        extra_fields.setdefault('is_staff', False)
+        extra_fields.setdefault('is_superuser', False)
+        return self._create_user(email, password, **extra_fields)
+
+    def create_superuser(self, email, password, **extra_fields):
+        extra_fields.setdefault('is_staff', True)
+        extra_fields.setdefault('is_superuser', True)
+
+        if extra_fields.get('is_staff') is not True:
+            raise ValueError('Superuser must have is_staff=True.')
+        if extra_fields.get('is_superuser') is not True:
+            raise ValueError('Superuser must have is_superuser=True.')
+
+        return self._create_user(email, password, **extra_fields)
+
+
+class User(AbstractBaseUser, PermissionsMixin):
+
+    email = models.EmailField('email address', unique=True, help_text=(
+        'メールアドレスを入力して欲しいっぴ'),)
+
+    nickname = models.CharField(verbose_name='ニックネーム', max_length=10, blank=True, null=True, help_text=(
+        '君の名は。'),)
```

Comment:
多分オートフォーマットで途中で改行されたのだと思うのですが，かえって見辛くなってしまっているので，中途半端に改行されてしまった時は1つ1つ改行するのがおすすめです。introductionも同様です
```suggestion
    nickname = models.CharField(
        verbose_name='ニックネーム',
        max_length=10,
        blank=True,
        null=True,
        help_text=('君の名は。'),
    )
```